.gitignore

```
@@ -62,3 +62,7 @@ pyglui/cygl/glew\.pxd
 pyglui/cygl/unhandled_glew\.h
 
 pip-*-build/
+pip-*-metadata/
+
+.venv
+.vscode/
```

Add newline at the end of file
```suggestion
.vscode/

```